src/components/utility/Typography/Typography.js

```diff
@@ -1,29 +1,35 @@
-import React from 'react';
+import React from "react";
+import styles from "./Typography.module.css";
 
 export const TypographyElements = {
-  "body1": "p",
-  "body2": "p",
-  "span": "span",
-  "h1Small": "h1",
-  "h1Large": "h1",
```

Comment:
See [comment on CSS](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/8/files#r1651710253) for discussion of why some kind of h1Large should remain

Reply:
![tablet](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/assets/31151391/9e5f0887-12c7-44f8-b7b3-95e12365d752)
![mobile](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/assets/31151391/bce674fd-d822-4254-adb8-1eac780ba26c)
![desktop](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/assets/31151391/52612726-5fc7-4d77-8e7c-df51979e0caa)

Reply:
h1large and h1small are the styles for the same text in 2 different device. We don't have the same style for mobile in the typography catalog

Reply:
<img width="715" alt="Screenshot 2024-06-26 at 12 28 51 PM" src="https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/assets/284712/dcbccf5b-a4ac-4ba9-be37-7be3d2f4a4f6">
The "About" and "Contact" headers are h1-small. They are 64px (tablet and desktop) but the home page header gets up to 100px (h1-large) on desktop, while remaining being 64px on tablet.